pkg/testing/integration_test.go

```diff
@@ -437,3 +465,58 @@ func pingAppAndWait(ctx context.Context, a app.DatabaseApp) error {
 	})
 	return err
 }
+
+// getServiceAccount return ServiceAccount object
```

Comment:
the comment is not useful, we can include why are we doing this when we are calling this. Let's remove this comment from here.